build/MSBuildExtensions.targets

```diff
@@ -131,6 +131,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <BundledNETCoreAppPackageVersion>$(_NETCoreAppPackageVersion)</BundledNETCoreAppPackageVersion>
     <BundledNETStandardTargetFrameworkVersion>$(_NETStandardTargetFrameworkVersion)</BundledNETStandardTargetFrameworkVersion>
     <BundledNETStandardPackageVersion>$(_NETStandardPackageVersion)</BundledNETStandardPackageVersion>
+    <NETCorePlatformsImplicitPackageVersion>$(MicrosoftNETCoreAppPackageVersion)</NETCorePlatformsImplicitPackageVersion>
```

Comment:
I don't think the Platforms package will always have the same version number as the NETCoreApp package.  Did you verify whether this will be the case with the runtime team?

Reply:
This is a problem. I think we should remove this property in CLI. And we should get this property in SDK derived from its dependency 
![image](https://user-images.githubusercontent.com/6993335/35073014-e2a98c52-fb9b-11e7-82d1-8135ef9ed316.png)

Reply:
PackageDefinitions won't work, This need to be added before PackageDefinitions is computed

Reply:
We are already getting the NETStandard.Library 2.0.1 from PackageDefinitions here in this file. Why does this need to be computed before PackageDefinitions in this file?

Reply:
"computed before PackageDefinitions" in during pack, ideally this should computed according to consumer project's runtime. Instead of predefined in CLI. the problem is if the user set _RuntimeFrameworkVersion_, and we will add the package version according to the runtime CLI carries. If RuntimeFrameworkVersion > CLI's runtime, there will be a package downgrade.

Reply:
Talked to Nick. For preview 1, we should get the version from PackageDefinitions and set it in build/MSBuildExtensions.targets.

After that, we need to design whether we want to keep `Microsoft.NETCore.Platforms` on Pack side or install side (add the reference in fake project). Or maybe we can ask NuGet for pack target  "add extra dependency" feature.

Also, the property name need to change. This require change on both SDK and CLI, I will send PR once this is merged. https://github.com/dotnet/cli/issues/8421

Reply:
Changed as discussed above